src/metabase/pulse/render/body.clj

```diff
@@ -512,14 +512,51 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(defn- largest-range
+  [{:keys [fingerprint]}]
+  (let [{mn :min mx :max} (get-in fingerprint [:type :type/Number])]
+    (- mx mn)))
```

Comment:
nit: `largest` made me think that this would be something like

```
(defn- largest-range
  [fingerprints] ;; plural
  (apply max
         (for [{:keys [min max] fingerprints]
            (- max min)))
```

Since it just takes the one fingerprint maybe plain ol' `range` is a better name?

Reply:
Fair point, though I might try to avoid shadowing clojure.core/range. I can make the fn name clearer for sure.